src/Uno.UI.RemoteControl.Host/Extensibility/Uno.Extensions.DependencyInjection/AttributeDataExtensions.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Uno.Extensions.DependencyInjection;
```

Comment:
Can we avoid using Uno.Extensions namespace to avoid collisions with Uno.Extensions

Reply:
I agree but currently all those kind of helpers are in the `Extensions` namespace in Uno. If we decide to start to use another one, what would be your suggestion ? `Helpers`? `Utils`?

cc. @jeromelaban @carldebilly

Reply:
Renamed to `Uno.Utils.DependencyInjection`. This is matches what we already have for `Uno.Utils.Clipboard`